server/app.js

```diff
@@ -31,8 +31,8 @@ app.use("/", indexRouter);
 app.use("/ping", pingRouter);
 app.use("/login", loginRouter);
 app.use("/signup", signupRouter);
-app.use("/find", findUserRouter);
-app.use("/delete", deleteUserRouter);
+app.use("/findDeleteUser", findDeleteUserRouter);
```

Comment:
```suggestion
app.use("/user", userRouter);
```
This is more extendable as we will probably want more options in the future